scripts/actions/mobskills/blackout.lua

```diff
@@ -0,0 +1,26 @@
+-----------------------------------
+-- Blackout
+-- Description: AoE Silence, Paralysis, Blind.
+-- Attack Type: Magical
+-- Shadows: Ignores
+-----------------------------------
+---@type TMobSkill
+local mobskillObject = {}
+
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    local power = math.random(10, 20)
+    local duration = math.random(60, 180)
+
+    xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.PARALYSIS, power, 0, duration)
+    xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.SILENCE, power, 0, duration)
```

Comment:
Silence power should be 1.

How were this numbers reached? Where was the info gotten from?

Reply:
Fixed Silence power

Reply:
> Tested on FFXI Server using the i117 as a PLD/NIN Tested on my server using the same gear as PLD/NIN with the same stats as in the FFXI server Fought NMs Over and over about 20 times until i got the approximated results. For mob skills effect, also used Bg-wiki as a reference. If there is a way to find out the numbers using capture please do let me know and i will go and do more.